File: src/app/src/main/java/com/onscreensync/tvapp/fragments/ImageMediaFragment.java
package com.onscreensync.tvapp.fragments;

import android.os.Bundle;

import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;

import com.onscreensync.tvapp.R;
import com.squareup.picasso.Picasso;

/**
 * A simple {@link Fragment} subclass.
 * Use the {@link ImageMediaFragment#newInstance} factory method to
 * create an instance of this fragment.
 */
public class ImageMediaFragment extends Fragment {

    private ImageView mediaImageView;

    public ImageMediaFragment() {
        // Required empty public constructor
    }

    public static ImageMediaFragment newInstance(String param1, String param2) {
        ImageMediaFragment fragment = new ImageMediaFragment();
        Bundle args = new Bundle();
        fragment.setArguments(args);
        return fragment;
    }

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        View view = inflater.inflate(R.layout.fragment_image_media, container, false);
        // Retrieve data from arguments
        Bundle bundle = getArguments();
        if (bundle != null) {
            String assetUrl = bundle.getString("assetUrl");
            mediaImageView = (ImageView) view.findViewById(R.id.media_image_asset);
            if(mediaImageView != null) {
                Picasso.get().load(assetUrl).into(mediaImageView);
            }
        }

        return view;
    }
}

File: src/app/src/main/java/com/onscreensync/tvapp/fragments/ImageMediaFragment.kt
package com.onscreensync.tvapp.fragments

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.onscreensync.tvapp.databinding.FragmentImageMediaBinding
import com.squareup.picasso.Picasso
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
class ImageMediaFragment : Fragment() {

    private var _binding: FragmentImageMediaBinding? = null
    private val binding get() = _binding!!

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        _binding = FragmentImageMediaBinding.inflate(inflater, container, false)
        return binding.root
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        arguments?.let { bundle ->
            val assetUrl = bundle.getString("assetUrl")
            if (!assetUrl.isNullOrEmpty()) {
                Picasso.get().load(assetUrl).into(binding.mediaImageAsset)
            }
        }
    }

    override fun onDestroyView() {
        super.onDestroyView()
        _binding = null
    }

    companion object {
        @JvmStatic
        fun newInstance(param1: String?, param2: String?) = ImageMediaFragment()
    }
}